### DISCUSSION TITLE
Wake-Up PIJuice when batterie is very low

### DISCUSSION THREAD

**Discussion by FrancoisL50** [PiSupply/PiJuice#1052] [Q&A]
Hello,
I have a PIJuice which powers a raspberry pi zero (v1.3). The Pijuice supplies 5v, from connector J3 (System switch on 2100mA), to an STM32 controller.
I have activity from the pijuice_cli interface, stopping the Raspbery, but not stopping the power supply to the System switch by setting the "Low Charge" value of the "System Event" menu with SYS_FUNC_HALT.
The "System Task" menu is configured with a value of 20% for "Wake-Up". The “Min Load” setting is at 75% to prioritize power to the STM32 in the event of a power outage.
When the battery is very low (below the Wake-Up value) and power is restored, randomly, the raspberry reboots and the system switch is not activated.
Is this normal behavior? Did I configure the Pijuice incorrectly?
Thank you for your help

**Comment by tvoverbeek** [comment]
J3 does *not* supply 5V, but VSYS (varying with the battery voltage).
The system switch setting is not saved. When the PiJuice (re)starts it is innitialized at 500mA
However you can trigger an user function at the system start event to set the system switch to the value you want (2100 mA).

**Reply by FrancoisL50** [reply]
Thank you for your reply.
You had already responded to this subject (System switch resets on startup #380).
Sorry, I confused P3 and J3.
And to restart the PI with the correct "Wake up" value when the battery is empty, you must activate the "restore" parameter in the "System Task" menu.

**Reply by FrancoisL50** [reply]
Hello,
I continue the tests with the wake-up function and I often encounter the following problem
After a power cut to the PIjuice, the raspberry does not wait for the value of the "wake-up" parameter to reboot. Sometimes it works, sometimes not... However, I revalidate the values ​​in the system task from the pjuice_cli interface
Can you give me an explanation for this behavior?
Thanks in advance
![image_2024-01-28_142307726](https://github.com/PiSupply/PiJuice/assets/155467927/61ef5746-d033-48b1-98a3-91f23c1bcd9a)
![image_2024-01-28_142331201](https://github.com/PiSupply/PiJuice/assets/155467927/fabd3636-f204-4e39-9b25-cd5f7e97d06c)
